tests/foreman/cli/test_activationkey.py

```diff
@@ -1581,3 +1581,63 @@ def test_positive_invalid_release_version(module_sca_manifest_org, module_target
         }
     )
     assert update_ak[0]['message'] == 'Activation key updated.'
+
+
+# -------------------------- MULTI-CV SCENARIOS -------------------------
+def test_positive_multi_cv_info(
+    session_multicv_sat, session_multicv_org, session_multicv_default_ak
+):
+    """Verify that multi content view environment details displays into hammer activation-key info commands output
+
+    :id: 6a1c3189-74f9-4a54-8579-f3b045870cd9
+
+    :steps:
+        1. Create two lifecycle environments and two content views, publish/promote to respective lce
+        2. Create activation key and update ak with multiple content view environments
+        3. Check that ak info displays 'Multi Content View Environment' and 'Content View Environments'
+
+    :expectedresults: AK info displays 'Multi Content View Environment' and 'Content View Environments'
+
+    :CaseImportance: Medium
+
+    :Verifies: SAT-27863
+    """
+    # Create two lifecycle environments
+    lces_list = [
+        session_multicv_sat.api.LifecycleEnvironment(organization=session_multicv_org).create()
+        for i in range(2)
+    ]
+    lce1 = lces_list[0]
+    lce2 = lces_list[1]
```

Comment:
We could expand like this for LCEs and CVs:
```suggestion
    lce1, lce2 = lces_list
```